tests/endtoend_tests/test_endtoend.py

```diff
@@ -318,6 +325,14 @@ def test_tesseract_serve_ports(built_image_name, port, docker_cleanup):
     cli_runner = CliRunner(mix_stderr=False)
     project_id = None
 
+    docker_host = os.environ.get("DOCKER_HOST", "")
+
+    if "-" in port and "podman" in docker_host:
+        pytest.skip(
+            "Podman does not support port ranges in compose."
+            "See https://github.com/containers/podman/issues/15111"
+        )
```

Comment:
How will this manifest when actual users try to use this feature? Big scary error I presume?

Reply:
If we can detect Podman reliably ahead of time (ideally not by parsing `DOCKER_HOST` but via `docker --version` or something like that) let's do it. Otherwise this should be acceptable for now.

Reply:
`docker` communicates with the Docker daemon via a socket, so we can "point" `docker` to a Podman daemon instead by overwriting `DOCKER_HOST`. But `docker` itself doesn't know which daemon it communicates with, as long as it is API compatible. `docker info` and `docker --version` don't show anything that indicates which daemon it is communicating with.

The only way I'm aware of to check if `docker` is communicating with Podman is to check the environment variable `DOCKER_HOST`.

Reply:
Should be fixed by not passing port ranges to `docker-compose` anymore (see latest commit).